hibernate-orm-modules/hibernate-orm-modules.gradle

```diff
@@ -39,11 +39,6 @@ ext {
 	fpackStagingDir = file( "target/featurepack" ) //Target build directory for the Feature Pack
 }
 
-if ( JavaVersion.current().isJava11Compatible() ) {
-	logger.warn( '[WARN] Skipping all tests for hibernate-orm-modules due to Gradle issues with JDK 11' )
-	test.enabled = false
-}
```

Comment:
Does it work with JDK 13? Otherwise this condition should be updated, not removed.

Reply:
no it doesn't

Reply:
it seems I found a solution